lib/vehicle.rb

```diff
@@ -1,18 +1,22 @@
 require 'date'
 
 class Vehicle
+  attr_accessor :registration_date
```

Comment:
No need for an attr_accessor here since you created a method of `set_registration_date` that handles changing this attribute.